src/main/java/website/frontrow/util/Point.java

```diff
@@ -87,6 +87,26 @@ public Point divide(double divisor)
     }
 
     /**
+     * Subtract a point from the current point.
+     * @param b Point to subtract.
+     * @return Result of subtraction.
+     */
+    public Point subtract(Point b)
+    {
+        return new Point(this.x - b.x, this.y - b.y);
+    }
+
+    /**
+     * Multiply both coordinates of a point by a number.
+     * @param b The number to multiply by.
+     * @return A point with both coordinates multiplied by b.
+     */
+    public Point multiply(double b)
```

Comment:
Not an abbreviation, just had no clue what to name the other point.
Because pretty much any name works.

The current name signifies is that in a * b, this is the value b.
Any suggestions?